docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 ** *User name*: admin
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
+
+== Memory testing requirements for installing on Red Hat OpenShift Local
+
+https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local] requires a minimum amount of memory to complete the {ProductShortName} analysis. The more memory, the quicker the analysis is completed. The table below describes the various amounts of memory required by OpenShift Local to complete the MTA binary analysis:
```

Comment:
@CarmiWisemon 
When installed on https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local], {ProductShortName} requires a minimum amount of memory to be able to complete its analysis. Adding memory above the required minimum makes the analysis process run faster. The table below describes the MTA performance with varying amounts of memory.